docs/repositories/files.md

```diff
@@ -25,34 +25,23 @@ Use the search box to filter the list and find specific files:
 
 ## File details
 
-Click a specific file to see more detailed analysis information for that file, including statistics on:
+Click a specific file to see more detailed analysis information for that file.
 
--   **Size:** Lines of code, source lines of code, and commented lines of code
--   **Structure:** Number of methods and ratio of source lines of code per number of methods
--   **Complexity:** Complexity and complexity per method
--   **Duplication:** Number of clones and duplicated lines of code
-
-The button **Ignore File** allows you to ignore the selected file on future Codacy analysis.
+The header of the file detail page displays the same code quality metrics as the Files page, along with an **Ignore file** link to ignore the file, and a link to view the file on your Git provider.
```

Comment:
We could keep the consistency with [other pages describing similar links](https://github.com/codacy/docs/blob/master/docs/repositories/pull-requests.md?plain=1#L22-L26) in the header. 

```suggestion
The header of the file detail page displays the same code quality metrics as the Files page, along with an **Ignore file** link to ignore the file, and a link to view the file on your Git provider.
```